gulpfile.js

```diff
@@ -113,7 +113,7 @@ class AddClosureTypeImport extends Transform {
 
 gulp.task('clean', () => del([DIST_DIR, 'closure.log']));
 
-gulp.task('closure', ['clean'], () => {
+gulp.task('closure', ['generate-externs'], () => {
```

Comment:
Is this change ok? @azakus?

Reply:
Reasoning for this change is that `closure-lint` would not properly pick up the new changes. Whenever we run `closure`, the externs should be up-to-date to generate the correct output. `generate-externs` in turn invokes `clean`, so that should still work.